packages/ui-components/src/components/collapser.ts

```diff
@@ -4,7 +4,9 @@
 import { Message } from '@lumino/messaging';
 import { ISignal, Signal } from '@lumino/signaling';
 import { Panel, PanelLayout, Title, Widget } from '@lumino/widgets';
-import { caretDownIcon, caretUpIcon } from '../icon';
+import { caretDownIcon } from '../icon';
+
+const collapsedClass = 'jp-Collapse-header-collapsed';
```

Comment:
Should we use all caps for consts to be consistent with other places. For example:
https://github.com/jupyterlab/jupyterlab/blob/3ecf75ac8eda5f11e55f318658577bba227ecdf6/packages/cells/src/collapser.tsx#L16

```suggestion
const COLLAPSED_CLASS = 'jp-Collapse-header-collapsed';
```